.taskcluster.yml

```diff
@@ -212,7 +212,7 @@ tasks:
                           features:
                               taskclusterProxy: true
 
-                          image: mozillareleases/taskgraph:decision-f19bfa3ae9dbc19190149445bfc38ad966b713336e44342c7643b3f7f1e0b023@sha256:ddcc7348db272885e8ea32825f698bfafa4727401099531cd8239d3d458b39aa
+                          image: mozillareleases/taskgraph:decision-c4ac262880970ca484105929e02dd12f00214d5f2603ab6ce4c0d17de5cd0280@sha256:2dd667994aa13fccdcdcede85c570a5eb4a5247e42875a9e95a135ef774ee469
```

Comment:
I'd maybe bump the checkout cache version name along with this (same sort of reasons as https://phabricator.services.mozilla.com/D196015#6501654, although maybe git is OK here)

Reply:
Ah good call. I've been neglecting this in other repos and there hasn't been an issue yet, but doesn't hurt.